apps/financial-aid/web-osk/src/utils/hooks/useApplication.ts

```diff
@@ -41,7 +41,12 @@ const useApplication = () => {
         const files = formatFiles(form.taxReturnFiles, FileType.TAXRETURN)
           .concat(formatFiles(form.incomeFiles, FileType.INCOME))
           .concat(formatFiles(form.otherFiles, FileType.OTHER))
-          .concat(formatFiles(form.taxReturnFromRskFile, FileType.TAXRETURN))
+          .concat(
+            formatFiles(
+              form.taxReturnFromRskFile,
+              form.taxReturnFromRskFile[0].type,
+            ),
```

Comment:
Very good question but I don't think so, because this calls a function `formatFiles` which maps over the files and returns them in a certain format. This array is initialized to [], so the array itself won't be undefined and if the list is empty then we are mapping over empty array and return nothing. 

However I think a better implementation of our files would be that we would have one filelist in the form and that list has the filetype like this does, instead of concatting here, but that is a latter task.